atuin/src/shell/atuin.bash

```diff
@@ -1,5 +1,7 @@
 ATUIN_SESSION=$(atuin uuid)
+ATUIN_INTERPRETER=bash
 export ATUIN_SESSION
+export ATUIN_INTERPRETER
```

Comment:
All integrations now should (SHOULD, not MUST) include `ATUIN_INTERPRETER` variable